build_tools/check-meson-openmp-dependencies.py

```diff
@@ -14,9 +14,15 @@
 
 
 def has_source_openmp_flags(target_source):
+    """Check if a source in the target has OpenMP flags"""
```

Comment:
Given these are one-line implementations, I think it's okay to not have a docstring.